.integration

```diff
@@ -1,4 +1,6 @@
 command = 
 settings_file_name = 
 default_schedule = 0 0/5 * * * ?
```

Comment:
is `read_from_stdout` and `default_schedule` ever make sense here? Could we remove it?

Reply:
Yes, they are useless here.
You can remove them at all
```
command = 
settings_file_name = 
default_schedule = 0 0/5 * * * ?
read_from_stdout = false
```